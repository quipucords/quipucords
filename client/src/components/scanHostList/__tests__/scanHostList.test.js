import React from 'react';
import configureMockStore from 'redux-mock-store';
import { shallow } from 'enzyme';
import ScanHostList from '../scanHostList';

describe('ScanHostList Component', () => {
  const generateEmptyStore = () => configureMockStore()({});

  it('should shallow render a basic component with an empty state status', () => {
    const store = generateEmptyStore();
    const props = {};
    const wrapper = shallow(<ScanHostList {...props} status="success" />, { context: { store } });

    expect(wrapper.dive()).toMatchSnapshot();
  });
});
