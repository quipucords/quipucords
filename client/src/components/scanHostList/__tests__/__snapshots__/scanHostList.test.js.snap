// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScanHostList Component should shallow render a basic component with a basic status 1`] = `
<EmptyState
  className=""
>
  <EmptyStateIcon
    className=""
    name="warning-triangle-o"
    type="pf"
  />
  <EmptyStateTitle
    className=""
  >
    Successful systems were not available, please refresh.
  </EmptyStateTitle>
</EmptyState>
`;
